config_vsc.py

```diff
@@ -24,6 +26,20 @@
 # vsc group
 kul_account_string_tier2 = '-A lpt2_vsc_test_suite'
 
+# Specify hortense access flag in order to run jobs
+# Flag is selected according to user group
+groups = os.popen('groups').read().strip().split()
```

Comment:
```suggestion
groups = [x.gr_name for x in grp.getgrall()]
```

Reply:
oh you're right, my bad.
then I prefer doing it like this:

```python
hortense_access_flag = ''
groups = [grp.getgrgid(x).gr_name for x in os.getgroups()]
for admingroup in ['astaff', 'badmin', 'gadminforever', 'lstaff']:
    if admingroup in groups:
        hortense_access_flag = f'-A {admingroup}'
        break
```

Reply:
looks better.

Reply:
committed